apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -44,7 +44,161 @@ const { freezeBillingReceipt } = require('@condo/domains/acquiring/utils/freezeB
 const get = require('lodash/get')
 const Big = require('big.js')
 const validate = require('validate.js')
-
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@core/keystone/errors')
+const { REQUIRED, NOT_UNIQUE, NOT_FOUND, DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const { WRONG_FORMAT } = require('../../common/constants/errors')
+const { MULTIPLE_ACQUIRING_INTEGRATION_CONTEXTS, RECEIPTS_ARE_DELETED, RECEIPTS_HAVE_NEGATIVE_TO_PAY_VALUE,
+    ACQUIRING_INTEGRATION_DOES_NOT_SUPPORTS_BILLING_INTEGRATION, RECEIPTS_HAS_MULTIPLE_CURRENCIES,
+    RECEIPT_HAS_DELETED_BILLING_INTEGRATION, BILLING_RECEIPT_DOES_NOT_HAVE_COMMON_BILLING_ACCOUNT_WITH_SERVICE_CONSUMER,
+    BILLING_INTEGRATION_ORGANIZATION_CONTEXT_IS_DELETED, ACQUIRING_INTEGRATION_CONTEXT_IS_MISSING,
+    ACQUIRING_INTEGRATION_IS_DELETED, RECEIPTS_CANNOT_BE_GROUPED_BY_ACQUIRING_INTEGRATION,
+    CANNOT_FIND_ALL_BILLING_RECEIPTS,
+} = require('../constants/errors')
+
+const errors = {
```

Comment:
can we move this object to separate file ?

Reply:
No, because errors contains path to mutation arguments which is declared in this module.